### ISSUE TITLE
THERE ARE NO ASSISTANT AUDIO DATA RESPONCE?

### ISSUE THREAD

**Issue by vanlnb** [google/aiyprojects-raspbian#447]
I set up a kit like introduction of v1 with rpi3b everything is ok until i run the demo, it can recognize my voice, but it only return text, no sound audio return, i flash again and use old credential that create long time ago and it's work, but when i flash again and and use a new credential: there no answer sound respond, i check the code and found that there's no data audio return when recognize. Any body have the same issue? please help me.

**Comment by manoj7410** [comment]
@phong31151812 Can you try with the latest sdcard image ? Image can be downloaded from : https://github.com/google/aiyprojects-raspbian/releases

**Comment by cray-** [comment]
Tested the new 2018-08-03 image on a rpi3b with v1 voicehat. Still no response from the demos. grpc demo correctly recognises commands, everything shows up on myactivity.google.com, check_audio works fine (although my recorded voice sounds very quiet). But not getting any audio responses.

Further to that, if I run assistant_library_demo I get the contents of a webpage piped to the terminal. Looks to be same as this issue: https://github.com/googlesamples/assistant-sdk-python/issues/274

**Comment by dmitriykovalev** [comment]
@cray That's weird, when did you setup your credentials? You are right, looks the same as  googlesamples/assistant-sdk-python#274, we'll double check everything.

**Comment by cray-** [comment]
@dmitriykovalev I used an existing assistant.json file from a project I created when the original v1 voicekit came out with MagPi in Australia last year. I also flashed a fresh 2018-08-03 image and created a new set of Oauth2 credentials on the same project. Results the same as above. 

I just created a brand new project, added the cloud API and created a new set of credentials as per instructions here: https://aiyprojects.withgoogle.com/voice-v1/#users-guide-1-setting-up-your-device
Then flashed a new 2018-08-03 image and used the newly created assistant.json file. Now everything seems to be working!
